rituals.py

```diff
@@ -254,6 +254,9 @@ async def mkRitual(req):
                     bs_proc, bs_port = await launchBBS()
                 if 'videoUrl' in slide:
                     videos.add(slide['videoUrl'])
+            elif slide['widget'] == 'Video':
+                if 'videoUrl' in slide:
```

Comment:
its possible that literally all slides should just check for 'videoUrl' here, instead of first checking if they're an appropriate Widget?